server.js

```diff
@@ -0,0 +1,80 @@
+const express = require("express");
+const bodyParser = require("body-parser");
+const nodemailer = require("nodemailer");
+const cors = require("cors");
+const path = require("path");
+const cluster = require("cluster");
+const numCPUs = require("os").cpus().length;
+
+require("dotenv").config();
+
+const PORT = process.env.port || 8080;
+const CLIENT_DISTRIBUTION_DIRECTORY = path.resolve(__dirname, "./build");
+const isDev = process.env.NODE_ENV !== "production";
+
+// Multi-process to utilize all CPU cores.
```

Comment:
Creating node cluster on machine to maximize performance and throughput on the machine.